src/nav/navActions.js

```diff
@@ -62,12 +62,8 @@ export const navigateToGroupDetails = (
   recipients: $ReadOnlyArray<UserId>,
 ): GenericNavigationAction => StackActions.push('group-details', { recipients });
 
-export const navigateToRealmInputScreen = (
-  // The `Object.freeze`` in the `:` case avoids a Flow issue:
-  // https://github.com/facebook/flow/issues/2386#issuecomment-695064325
-  args: {| realm?: URL, initial?: boolean |} = Object.freeze({}),
```

Comment:
Callers haven't passed `initial` for a long time, and the last
    caller that passed `realm` was removed in the previous commit.

In "a recent commit", perhaps? I think the previous commit doesn't involve a call to this function (and in fact it provides `initial: true`.)